schema/diasource.avsc

```diff
@@ -0,0 +1,176 @@
+				{"name": "trailCov04", "type": "float"},
+				{"name": "trailCov05", "type": "float"},
+				{"name": "trailCov06", "type": "float"},
+				{"name": "trailCov07", "type": "float"},
+				{"name": "trailCov08", "type": "float"},
+				{"name": "trailCov09", "type": "float"},
+				{"name": "trailCov10", "type": "float"},
+				{"name": "trailCov11", "type": "float"},
+				{"name": "trailCov12", "type": "float"},
+				{"name": "trailCov13", "type": "float"},
+				{"name": "trailCov14", "type": "float"},
+				{"name": "trailCov15", "type": "float"}
+					]},
+			"null"]},
+		{"name": "trailFluxSigma", "type": ["float", "null"]},
+		{"name": "trailLengthSigma", "type": ["float", "null"]},
+		{"name": "trailAngleSigma", "type": ["float", "null"]},
+		{"name": "trailFlux_trailLength_Cov", "type": ["float", "null"]},
+		{"name": "trailFlux_trailAngle_Cov", "type": ["float", "null"]},
+		{"name": "trailLength_trailAngle_Cov", "type": ["float", "null"]},
+		{"name": "trailLength", "type": ["float", "null"]},
+		{"name": "trailAngle", "type": ["float", "null"]},
+		{"name": "trailLnL", "type": ["float", "null"]},
+		{"name": "trailChi2", "type": ["float", "null"]},
+		{"name": "trailN", "type": ["int", "null"]},
+		{"name": "dipMeanFlux", "type": ["float", "null"]},
+		{"name": "dipFluxDiff", "type": ["float", "null"]},
+		{"name": "dipRadec", "type": [{"type": "record",
+			"name": "dipRadec", "fields": [{"name": "dipRa", "type": "double"}, {"name": "dipDec", "type": "double"}]},
+			"null"]},
+		{"name": "dipLength", "type": ["float", "null"]},
+		{"name": "dipAngle", "type": ["float", "null"]},
+		{"name": "dipCov", "type": [{"type": "record",
+			"name": "dipCov", "fields": [
+				{"name": "dipCov01", "type": "float"},
+				{"name": "dipCov02", "type": "float"},
+				{"name": "dipCov03", "type": "float"},
+				{"name": "dipCov04", "type": "float"},
+				{"name": "dipCov05", "type": "float"},
+				{"name": "dipCov06", "type": "float"},
+				{"name": "dipCov07", "type": "float"},
+				{"name": "dipCov08", "type": "float"},
+				{"name": "dipCov09", "type": "float"},
+				{"name": "dipCov10", "type": "float"},
+				{"name": "dipCov11", "type": "float"},
+				{"name": "dipCov12", "type": "float"},
+				{"name": "dipCov13", "type": "float"},
+				{"name": "dipCov14", "type": "float"},
+				{"name": "dipCov15", "type": "float"}
+					]},
+			"null"]},
+		{"name": "dipLnL", "type": ["float", "null"]},
+		{"name": "dipChi2", "type": ["float", "null"]},
+		{"name": "dipN", "type": ["int", "null"]},
+		{"name": "fpFlux", "aliases": ["totFlux"], "type": ["float", "null"]},
+		{"name": "fpFluxSigma", "aliases": ["totFluxErr"], "type": ["float", "null"]},
+		{"name": "diffFlux", "type": ["float", "null"]},
+		{"name": "diffFluxSigma", "type": ["float", "null"]},
+		{"name": "fpSky", "aliases": ["fpBkgd"], "type": ["float", "null"]},
+		{"name": "fpSkySigma", "aliases": ["fpBkgdErr"], "type": ["float", "null"]},
+		{"name": "E1", "type": ["float", "null"]},
```

Comment:
Are E1 and E2 ellipticity parameters?  Does this make them redundant with Ixx, Iyy, and Ixy?